prep_data/tabular_data/train_featurize_train_tabular_data.ipynb

```diff
@@ -0,0 +1,673 @@
+{
```

Comment:
<p>Add a conclusion section suggesting what to do next now that you've processed the model and trained a model? Such as resources for how to deploy/predict?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3377/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
As this is a data preparation notebook, the main focus on the data preparation. The model training is really just a verification that our data preparation worked.

Reply:
IMO, prep_data is a weird folder for this reason

Reply:
Yeah, exactly! Like is it our job to showcase SageMaker capabilities or teach actual ML? Definitely worth a wider discussion for sure